PENDING.md

```diff
@@ -53,6 +53,7 @@ FEATURES
   * [cli] \#2047 Setting the --gas flag value to 0 triggers a simulation of the tx before the actual execution. The gas estimate obtained via the simulation will be used as gas limit in the actual execution.
   * [cli] \#2047 The --gas-adjustment flag can be used to adjust the estimate obtained via the simulation triggered by --gas=0.
   * [cli] \#2110 Add --dry-run flag to perform a simulation of a transaction without broadcasting it. The --gas flag is ignored as gas would be automatically estimated.
+  * [cli] \#966 Add --generate-only flag to build an unsigned transaction and write it to STDOUT.
```

Comment:
I think we should also add a note about `generate_only` to the send REST endpoint.

Reply:
Done, thanks